bundles/org.openhab.binding.openthermgateway/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -322,38 +342,378 @@
 	</channel-type>
 
 	<channel-type id="burnerhours">
-		<item-type>Number:Time</item-type>
+		<item-type>Number:Dimensionless</item-type>
```

Comment:
What's the reason for this change?

Reply:
It's not a Time data type, it's just a number of hours (integer value).

Reply:
Numer:Time represents a timespan (e.g. in hours). You might have mixed it up with with the Item type DateTime, which represents a date.

By using Units of Measure (Number:Time), the user can choose the desired unit and the framework takes care for the correct conversion.

Reply:
Done